js/script.js

```diff
@@ -0,0 +1,49 @@
+$(document).ready(function(){
+	var images = []
+	var interval = -1
+	var index = 0
+
+ 	function getImages(subject) {
+ 		var url = "http://www.reddit.com/search.json?q="+subject+"nsfw:no";
+ 		$.get(url, {
+ 			}).done(function(data){
+ 			 images = data.data.children.map(function(child){
+ 				return child.data;
+ 				}).filter(function(child) {
+ 					return child.preview != undefined;
+ 				}).map(function(child) {
+ 					return child.preview.images[0].source.url
+ 				})
+ 				console.log(images);
+ 			});
+ 			var filtered = searchResult.filter(function (filter){
```

Comment:
This function can be taken out, since you're already filtering the results in the `.filter()` function above. Also, there's no `searchResult` variable in existence.